test/support/html_test_helpers.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Phoenix.LiveView.HtmlTestHelpers do
+  @moduledoc false
+
+  require EasyHTML
+
+  defmacro sigil_X({:<<>>, _, [binary]}, []) do
```

Comment:
I must use a single-letter sigil to be compatible with elixir < 1.5. The X is completely arbitrary, happy to change to a different one.

Reply:
Note that EasyHTML defines an `HTML` sigil, but I assume we are doing our own here so that we can support older elixirs in the test matrix?

Reply:
derp you literally answered right above this! Sounds good. Note that you'll need a lowercase definition to support interpolation. This is all fine because it's just the test suite so we aren't exposing this to anyone for use.